Produlytics/ui/back-end/src/main/java/it/deltax/produlytics/uibackend/admins/devices/adapters/AdminCharacteristicAdapter.java

```diff
@@ -91,7 +91,7 @@ public int insertByDevice(int deviceId, NewCharacteristic characteristic) {
 			characteristic.sampleSize().isPresent() ? characteristic.sampleSize().getAsInt() : null,
 			false
 		));
-		return entity.getId();
+		return this.repo.findByDeviceIdAndName(deviceId, characteristic.name()).get().getId();
```

Comment:
Considera che non ho testato con `entity.getId()` dopo aver cambiato il `@GeneratedValue` in `CharacteristicEntity`.

Reply:
Ma infatti sti cambiamenti volevo metterli in un altro branch. Più che altro prima dovremmo chiudere questo. Forse avremmo dovuto chiuderlo prima dell'integrazione, così almeno avevamo tutto in developement.

Reply:
Quindi faccio il revert di quel commit e faccio su un altro branch?

Reply:
Per ora lascia così, magari questo fine settimana ci do un'occhiata e vedo se è possibile semplificarlo.